### PR TITLE
Update Network_Base.cpp

### PR DESCRIPTION
**Pull request by edvinsmineikis**
Made a few things more tidy.

### CODE REVIEW

File: KNN/Network_Base.cpp [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-
-
 Network_Base::Network_Base(size_t dz, size_t iz, size_t oz, unique_ptr<float[]> &data_ref) :
 	data_size(dz), input_size(iz), output_size(oz)
 {
 	data.reset(data_ref.get()); 
 
-	if (data_size - output_size == input_size) {  //if data size and input layer are the same size each neuron has just one input
+	if (data_size - output_size == input_size)
+	{  //if data size and input layer are the same size each neuron has just one input
```
Comment (SPauly) [inline KNN/Network_Base.cpp:9]: I see why you did these changes! I still think these are personal preferences and would not like to merge this. I was trying to use the [1TBS  ](https://en.wikipedia.org/wiki/Indentation_style#Variant:_1TBS_(OTBS)) I probably didn't follow through with it but would like to keep the brackets as they were.

File: KNN/Network_Base.cpp [hunk 2]
```diff
@@ -27,10 +28,10 @@ Network_Base::Network_Base(size_t dz, size_t iz, size_t oz, unique_ptr<float[]>
 	}
 
 	//Lernen
-
```
Comment (SPauly) [inline KNN/Network_Base.cpp:30]: This looks good let's keep that!

File: KNN/Network_Base.cpp [hunk 3]
```diff
@@ -27,10 +28,10 @@ Network_Base::Network_Base(size_t dz, size_t iz, size_t oz, unique_ptr<float[]>
...
-	while (output_layer[0].adjust_weights(data[data_size - 1], input_size) > 5.28419e-06) {	};
+	while (output_layer[0].adjust_weights(data[data_size - 1], input_size) > 5.28419e-06)
+	{
+	};
```
Comment (SPauly) [inline KNN/Network_Base.cpp:33]: Same here with the brackets. There's nothing wrong with this code but I'd love to keep it all in one style

File: KNN/Network_Base.cpp [hunk 4]
```diff
@@ -27,10 +28,10 @@ Network_Base::Network_Base(size_t dz, size_t iz, size_t oz, unique_ptr<float[]>
...
 	cout << "Found solution" << output_layer[0].output_y() << endl;
-
```
Comment (SPauly) [inline KNN/Network_Base.cpp:33]: Yeah let's reduce some unnecessary line breaks :)

**Review by SPauly — CHANGES_REQUESTED** [review]
I don't see anything wrong with these changes but I would like to stick to the 1TBS[](https://en.wikipedia.org/wiki/Indentation_style#Variant:_1TBS_(OTBS)). This is just personal preference and I'm excited to hear your opinion on that! 
Anyways thanks for having a look on this repository! This was just for some testing when I read a book and I will probably get back at it in some weeks... Currently working on a different Project.

**Comment by edvinsmineikis** [comment]
I want to learn about C++ and decided I should try and fix open issues on github to start learning. This is just a little introduction for me to learn about how GitHub works in general.
Otherwise, I styled it that way because I followed the guidelines of Notepad++ github, they have a very nice tutorial on how to style C++ content. Take a look: https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md

If you let me apply these changes, I can do a little work to style all files of this repository according to the manual. Just a little work for me to catch up to speed on how things work here. I would greatly appreciate it!

**Comment by SPauly** [comment]
Sounds good! I currently want to get back into GitHub too, which is why I greatly appreciate this opportunity here to get into an exchange with a fellow coder!  
Anyhow - I took a look into the Notepad++ Contribution file. As I said there are different types of indentation styles and I guess there is no wrong or right there. Which to choose depends on the developer or the existing project. In this scenario, I chose to more or less follow these instructions here since this will be the next project to work on: https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md 
I also provided these links in Issue #1 I guess. 

Still, I really appreciate your offer :) So here's an idea. I would really like to stick to the 1TBS and would like to implement it in the next weeks so if you want to use the Notepad++ Style it would probably be easier if you took a different project. But if you would like we can work together on some repository - maybe even a new one where we can start from scratch and decide which indentation style to use - in order to train to work with Github in a team. How does that sound?

**Comment by edvinsmineikis** [comment]
I suggest we exchange contacts on some platform and have a discussion. I have a lot of comments regarding myself and will totally have something to say about any further offers you make. Either way, it would be very appealing to contribute together in a project, new or existing.
You can add me on Discord. If there is some other platform you want to use, we can use that.

**Comment by SPauly** [comment]
Sounds good. Will reach out once I'll find the time to start a project together. I'm currently at work in a private repository and will just keep working at this rep for fun. So if you'd still like to contribute to this repository in the meantime I would highly appreciate it if you use 1TBS :) 
GG

**Pull request closed without merging**